src/pages/Tags/Tags.tsx

```diff
@@ -158,11 +158,11 @@ const Tags: FC = () => {
             tagsList={tagsList}
             type={type}
             isEditable={
-              type !== TagTypes.Skills ||
-              !includes(
-                userPrivileges,
-                Privileges.EditTag || Privileges.AddTag || Privileges.DeleteTag
-              )
+              type !== TagTypes.Skills
+                ? includes(userPrivileges, Privileges.DeleteTag) ||
+                  includes(userPrivileges, Privileges.AddTag) ||
```

Comment:
I might recommend getting this logic out of the inline prop since it is already several lines long.

Reply:
Changed it